app/processing/organisation/organisation-schema.js

```diff
@@ -0,0 +1,14 @@
+const Joi = require('joi')
+
+module.exports = Joi.object({
+  sbi: Joi.number().integer().required(),
+  addressLine1: Joi.string().required(),
+  addressLine2: Joi.string().allow('').allow(null).default(''),

```

Comment:
I think it might be safe to only `require` `addressLine1` and allow others  to be optional with `default('')`

Reply:
Replicated structure of statement-data service here

Reply:
Does not match `statement-data` schema: https://github.com/DEFRA/ffc-pay-statement-data/blob/main/app/publishing/organisation/schema.js